python/xgbserver/xgbserver/xgboost_model_repository.py

```diff
@@ -20,6 +20,12 @@ class XGBoostModelRepository(KFModelRepository):
     def __init__(self, model_dir: str = MODEL_MOUNT_DIRS, nthread: int = 1):
         super().__init__(model_dir)
         self.nthread = nthread
+        for name in os.listdir(model_dir):
```

Comment:
This bit of code is quite useful and used in a few locations.

Should it be a method on `KFModelRepository`? Something like `load_models_from_dir(dir: string)`